### DISCUSSION TITLE
Welcome to pyplaycricket Discussions!

### DISCUSSION THREAD

**Discussion by ewanharris12** [ewanharris12/pyplaycricket#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by XCIDesign** [comment]
Great work Ewan! I was in the process of creating something very very similar but then I came across this and realised I didn't need to duplicate your efforts.

One small improvement you could implement is to use the 'from_entry_date' parameter when using the match summary api. You could default this None, but if it is provided it allows uses to save the match summaries from a particular season and then just check for any that have been updated since the last check - rather than pulling all matches from the whole season each time the API call is made.
